manifests/noderesourcetopology/scheduler-config.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: kubescheduler.config.k8s.io/v1beta1
+apiVersion: kubescheduler.config.k8s.io/v1beta2
```

Comment:
The pluginConfig for `NodeResourceTopologyMatch' also needs to be updated to remove the kubeconfigpath.

Reply:
done.